### DISCUSSION TITLE
Is there a Release Schedule for Perspective?

### DISCUSSION THREAD

**Discussion by yuj2i0** [perspective-dev/perspective#2794] [General]
Looking to upgrade my perspective version, to include this fix
https://github.com/finos/perspective/pull/2784

Any rough timeline when will 3.1.1 be released?

Thank you

**Comment by texodus** [comment]
There is no release schedule, we release Perspective when it is ready. [Contact us at Prospective.co](mailto:hello@prospective.co) if you need enterprise support.
